src/styles/blocks/stars1.css

```diff
@@ -0,0 +1,11 @@
+.stars--1 :nth-child(-n + 1)::after {
```

Comment:
You don't need pseudo elements in this task, just change background-image to child elements to star-active

Reply:
Yes, but the task said that I should use it

"Yellow stars are added with **container modifier + pseudo-selector** (NO extra classes)"

Reply:
maybe I'm wrong understanding something?

Reply:
We already have `nth-child `pseudo-class so `after` is redundant here

Reply:
okey. `nth-child` it is also a pseudo class.